lib/bolt/outputter/human.rb

```diff
@@ -174,11 +174,22 @@ def print_result(result)
           if result.message?
             @stream.puts(remove_trail(indent(2, result.message)))
           end
-
           case result.action
           when 'command', 'script'
             safe_value = result.safe_value
-            @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+            if safe_value["merged_output"]
+              @stream.puts(indent(2, safe_value['merged_output'])) unless safe_value['merged_output'].strip.empty?
+
+            else # output stdout or stderr
+              unless safe_value['stdout'].nil? || safe_value['stdout']&.strip&.empty?
+                @stream.puts(indent(2, "STDOUT:"))
+                @stream.puts(indent(4, safe_value['stdout']))
+              end
+              unless safe_value['stderr'].nil? || safe_value['stderr']&.strip&.empty?
```

Comment:
```suggestion
              unless safe_value['stderr'].nil? || safe_value['stderr'].strip.empty?
```

Ditto.

Reply:
Is that because the nil check means we don't need the safe operators anymore?